src/cms-content/contact/index.ts

```diff
@@ -0,0 +1,55 @@
+import { sanitizeID, Markdown, TocItem } from '../utils';
+import content from './contact.json';
+
+interface ContactUsSection {
+  sectionId: string;
+  header: string;
+  body: Markdown;
+}
+
+interface ContactUsContent {
+  header: string;
+  intro?: Markdown;
+  sections: ContactUsSection[];
+}
+
+const sanitizeSection = (section: ContactUsSection): ContactUsSection => ({
+  sectionId: sanitizeID(section.sectionId),
+  header: section.header,
+  body: section.body,
+});
+
+const sanitizeContent = (content: ContactUsContent): ContactUsContent => ({
+  header: content.header,
+  intro: content?.intro || '',
+  sections: content.sections.map(sanitizeSection),
+});
+
+export const contactUsContent = sanitizeContent(content as ContactUsContent);
+
+export const sidebarItems: TocItem[] = [
+  {
+    to: '/contact',
+    label: 'Contact us',
+    items: contactUsContent.sections.map(section => ({
+      to: `/contact#${section.sectionId}`,
+      label: section.header,
+    })),
+  },
+];
+
+interface Page<T> {
+  pageTitle: string;
+  pageDescription: string;
+  canonicalUrl: string;
+  sidebarItems: TocItem[];
+  content: T;
```

Comment:
what does T stand for here

Reply:
oh, this was an experiment - The idea was to create a generic type for pages that we could reuse for all the CMS pages

```tsx
const ContactUsPage: Page<ContactUsContent> = {
  pageTitle: '...',
  // ...
  content: contactUsContent
}

const DonatePage: Page<DonateContent> = {
  pageTitle: '...',
  // ...
  content: donateContent
}
```

if we establish this as a standard for the pages, we could ensure that all the pages have metatags and sidebar items... but I think it might be overengineering a bit, what do you think?